modules/swagger-codegen/src/main/resources/lua/configuration.mustache

```diff
@@ -0,0 +1,43 @@
+{{>partial_header}}
+package {{packageName}}
+
+import (
+	"net/http"
+)
+
+const ContextOAuth2 		int = 1
+const ContextBasicAuth 		int = 2
+const ContextAccessToken 	int = 3
+const ContextAPIKey 		int = 4 
+
+type BasicAuth struct {
+	UserName      string            `json:"userName,omitempty"`
+	Password      string            `json:"password,omitempty"`	
+}
+
+type APIKey struct {
+	Key 	string
+	Prefix	string
+}
+
+type Configuration struct {
```

Comment:
Yes, and I'm not saying it's a must. My suggestion is to make it more consistent with other language generator. We can put it as a day 2 requirement for reconsideration.